lib/msf/core/db_manager/service.rb

```diff
@@ -120,7 +112,6 @@ def report_service(opts)
     end
 
     if (service and service.changed?)
-      msf_import_timestamps(opts,service)
```

Comment:
This can be deleted as the code here: https://github.com/rapid7/metasploit-framework/pull/19211/files#diff-27a6a9db37663d29e726020010f0d2264510f47f31220d80caabe593e79d2286R90-R96:~:text=service.state-,opts,-.each%20%7B
```ruby
    opts.each { |k,v|
      if (service.attribute_names.include?(k.to_s))
        service[k] = ((v and k == :name) ? v.to_s.downcase : v)
      elsif !v.blank?
        dlog("Unknown attribute for Service: #{k}")
      end
    }
```
iterates over all the option key-value pairs and assigns them to the service.